samples/notebookinstance_lifecycle_config/README.md

```diff
@@ -0,0 +1,37 @@
+# Notebook Instance Lifecycle Configuration sample
+
+## Prerequisites
+
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Create a Notebook Instance Lifecycle Configuration
+This command creates a Notebook Instance Lifecycle configuration.
+Note: The onCreate and onStart fields must be in base64.
+
+In the example crd the value of the onCreate field decodes to A="5" and the value of the onStart field decodes to:<br />
```

Comment:
In the example yaml what are the values of `onCreate` and `onStart` ?
Also can you explain how you got the base64 values for those

Reply:
A="5" and O="1" seem so random and don't make much sense 

can we just have `echo "Entering onStart"` and `echo "Entering onCreate"`

Reply:
done